es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/ElasticSearchDAOV6.java

```diff
@@ -91,62 +91,45 @@
 @Singleton
 public class ElasticSearchDAOV6 extends ElasticSearchBaseDAO implements IndexDAO {
 
-    private static final Logger logger = LoggerFactory.getLogger(ElasticSearchDAOV6.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ElasticSearchDAOV6.class);
 
     private static final String WORKFLOW_DOC_TYPE = "workflow";
-
     private static final String TASK_DOC_TYPE = "task";
-
     private static final String LOG_DOC_TYPE = "task_log";
-
     private static final String EVENT_DOC_TYPE = "event";
-
     private static final String MSG_DOC_TYPE = "message";
 
-    private static final String className = ElasticSearchDAOV6.class.getSimpleName();
-
     private static final int RETRY_COUNT = 3;
-
     private static final int CORE_POOL_SIZE = 6;
-
     private static final long KEEP_ALIVE_TIME = 1L;
-
     private static final int UPDATE_REQUEST_RETRY_COUNT = 5;
 
-    private String workflowIndexName;
-
-    private String taskIndexName;
-
-    private String eventIndexPrefix;
+    private static final String className = ElasticSearchDAOV6.class.getSimpleName();
 
+    private final String workflowIndexName;
+    private final String taskIndexName;
+    private final String eventIndexPrefix;
     private String eventIndexName;
-
-    private String messageIndexPrefix;
-
+    private final String messageIndexPrefix;
```

Comment:
I see the structure of the indices is more complicated than I suspected. Is that documented somewhere?

Reply:
In ES6, each index corresponds to one document type.